index.html

```diff
@@ -0,0 +1,148 @@
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>PORTFOLIO MOBILE VERSION</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./main.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+
+    <div class="container">
+
+        <div class="card">
+            <div class="card__header">
+                <img src="./images/Portfolio.png" alt="card__image" class="card__image" width="600">
+            </div>
+            <div class="card__body">
+                <span class="tonic">Tonic</span>
+                <h5><b>CANOPY</b> <span>&#183;</span> Back end Dev <span>&#183;</span> 2015</h5>
+                <p class="paratext">A daily selection of privately personalised reads no accounts or signups required</p>
+            </div>
+            <div class="card__footer">
+                <div class="user">
+                    <div class="user__info">
+                        <Button type="button" class="btn">html</Button>
+                        <Button type="button" class="btn">css</Button>
+                        <Button type="button" class="btn">javascript</Button>
+                        <h5><button type="button" class="btn-b">See Project</button></h5>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="card__header">
+                <img src="./images/Portfolio2.png" alt="card__image" class="card__image" width="600">
+            </div>
+            <div class="card__body">
+                <span class="tonic">Tonic</span>
+                <h5><b>CANOPY</b> <span>&#183;</span> Back end Dev <span>&#183;</span> 2015</h5>
+                <p class="paratext">A daily selection of privately personalised reads no accounts or signups required</p>
+            </div>
+            <div class="card__footer">
+                <div class="user">
+                    <div class="user__info">
+                        <Button type="button" class="btn">html</Button>
+                        <Button type="button" class="btn">css</Button>
+                        <Button type="button" class="btn">javascript</Button>
+                        <h5><button type="button" class="btn-b">See Project</button></h5>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="card__header">
+                <img src="./images/Portfolio3.png" alt="card__image" class="card__image" width="600">
+            </div>
+            <div class="card__body">
+                <span class="tonic">Tonic</span>
```

Comment:
- [ ] Kindly use a header tag `<hx>` `(h2, h3, ... , h6)` here as this is a work title instead of a `<span>`